src/test/java/seedu/address/testutil/AddressBookBuilder.java

```diff
@@ -37,4 +37,15 @@ public AddressBookBuilder withPerson(Person person) {
     public AddressBook build() {
         return addressBook;
     }
+
```

Comment:
Do you think we should place this function somewhere else? The builder classes generally create the Doctor/Patient/Addressbook class, with the build method. The get addressBook methods are in typical doctors and patients classes. In this case, should we create a new Typical person class to place this method inside instead of putting this function here?